client/index.html

```diff
@@ -1,31 +1,37 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta name="charset" content="utf-8">
     <title>Q</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" type="image/png" href="/favicon.png">
-    <base href="/">
+    <base href="https://localhost:9000/proxy/q-editor/">
     <!-- HEAD_MARKUP -->
   </head>
+
   <body aurelia-app="main">
     <!-- BODY_START_MARKUP -->
     <div style="position: absolute; top: calc(50% - 20vh); left: calc(50% - 15vh); width: 30vh; width: 30vh;">
       <svg viewBox="0 0 172 172">
-        <path d="M63.9,22.7l7.7-2.2c0,0,34.3-9.8,55.6,18.3c20.9,27.6,17,73.4-3.6,91.4c1.5,2.4,6,13,17.9,12.4c-10.8,4.7-21.4,4.3-28.2-9.4c0,0,20.3-15.7,17-53.5c-2.8-31.7-18.3-48.2-36.9-54.8C78.8,20.2,67.8,22.2,63.9,22.7z M130.4,161.4l10.2-3c9.1-3.5,15.4-14,19.2-23.4l-10.2,3C147.4,144.1,140.1,158.1,130.4,161.4z M48.7,70.1c-1.1,20.4,1.6,35.7,5.6,43.8l1.3,1.2c0,0,6.5-9,6-12.2c-3.1-11.9-4-29.2-1.5-43.5c2.4-13.6,9.2-24,15.7-27.1C61.3,33,49.9,48.1,48.7,70.1z M70.8,97.3c1.1-0.2,5.5-0.9,11.1,0.5c8.9,2.3,16.5,10.7,22.5,21c2.6-1.8,4.9-10.7,3.5-12.6c-1.4-1.9-5.9-8-13.6-10.8c-7.8-2.8-14.1-1-14.1-1L70.8,97.3z M81.1,131.6c6,0.6,11.5-1.1,13.7-2.8c-1.9,0.2-6.6-0.1-12.1-1.7c-5.2-1.5-9.2-4.6-11.1-6.6c0.3-8.3,4.6-11.7,6.3-12.4c-2.2,0.1-8.3,0.2-12.6,4.6c-4,4.1-3.9,10.9-3.9,10.9C67.6,129.6,73.9,131,81.1,131.6z"/>
-        <path d="M126.9,147c-0.4,0-0.7-0.1-1.1-0.1c-6.4-0.7-10.8-4.5-15.9-13.1c11.7-10.4,17.7-27.2,17.6-47.1c-0.3-16.9-4.9-32.4-13.4-43.4C105.5,32.1,93.2,25,77.9,23.7C43,20.8,27.7,46.2,28,80.2c0.4,36.4,22.1,60.4,49.5,62.7c7.8,0.7,14.6-0.2,21.4-2.7c6,11.6,12.6,20.7,23.5,21.6c9.8,0.8,18.4-9.9,22.8-20.5C138.5,145.8,134.5,147.6,126.9,147z M77.9,133.3c-8.3-0.7-14.7-3.8-19.7-8.6c0.1-10.1,6.3-16.7,14.9-16c10.7,0.9,15.5,11.1,20.7,21.5C89,132.9,84.1,134,77.9,133.3z M102.5,121.7c-8.3-13-15.1-21.9-27.4-22.9c-11.3-0.9-18.8,5.3-22.3,19.3c-5.9-11.7-7.4-24.3-7.6-37.1c-0.3-24.9,6.3-50.2,32.6-48c25.4,2.1,31.6,28,31.8,52.6C109.7,98.7,108.2,111.4,102.5,121.7z"/>
+        <path
```

Comment:
Saving that image separately on disk and then embed it would be nice, so developers actually see what kind of svg that is 😁 .

Reply:
Not something that was changed now though, only the autoformatter doing it's thing :-)
It's the Q logo btw.

Reply:
> It's the Q logo

I had to manually put it into a file to see that when I checked out the project the first time. 😁

Reply:
haha, I can decode SVG in my brain :-). Forgot that not everybody is able to do that.